bcda/models/models_test.go

```diff
@@ -25,3 +28,33 @@ func (s *ModelsTestSuite) TestJobStatusMessage() {
 	j = Job{Status: JobStatusCompleted, JobCount: 25, CompletedJobCount: 25}
 	assert.Equal(s.T(), string(JobStatusCompleted), j.StatusMessage())
 }
+
+func (s *ModelsTestSuite) TestACOBlacklist() {
+	blackListValues := Termination{
+		TerminationDate: time.Date(2020, time.December, 31, 23, 59, 59, 0, time.Local),
+		CutoffDate:      time.Date(2020, time.December, 31, 23, 59, 59, 0, time.Local),
+		BlacklistType:   Involuntary,
+	}
```

Comment:
Should we also add a case for Voluntary?

Reply:
Added tests for Voluntary and Limited.